test/test.typ

```diff
@@ -620,7 +625,7 @@ def floyd_warshall(G):
 
 #code(
```

Comment:
This test seems to have failed. But if you can't figure it out, don't bang your head over it. The approach used for `code` is currently outdated, and this is a niche test case anyway. Yet another thing to update when improved tables arrive.